pkg/actuators/machine/actuator.go

```diff
@@ -362,9 +353,19 @@ func (a *Actuator) Update(context context.Context, cluster *clusterv1.Cluster, m
 	// Parent controller should prevent this from ever happening by calling Exists and then Create,
 	// but instance could be deleted between the two calls.
 	if len(instances) == 0 {
-		glog.Warningf("attempted to update machine but no instances found")
+		glog.Warningf("attempted to update machine but no running instances found")
+
+		// check for all non-terminated instances, not just running
+		existingInstances, err := a.getExistingMachineInstances(cluster, machine)
+		if err != nil {
+			glog.Errorf("Error getting existing instances: %v", err)
+		} else {
+			if len(existingInstances) > 0 {
+				glog.Errorf("Found non-running instances with same machine name, administrative action may be required.")
```

Comment:
Existing instances include running instance. So, claiming to found non-running instance is misleading.